accredible-academy-theme.php

```diff
@@ -161,7 +161,7 @@ public static function issue_certificates_automatically() {
 
 				$issue = true;
 				foreach ($existing_certificates as $key => $certificate) {
-					if($certificate->recipient->email == $user->user_email){
+					if($certificate->recipient->email == strtolower( $user->user_email )){
```

Comment:
I could not confirm if this method is still being used, but this change would be necessary to make it work with emails that include uppercases.